test/integration/RedisResourceLockerIntegration.test.ts

```diff
@@ -0,0 +1,157 @@
+import type { HttpHandler, Initializer, ResourceStore } from '../../src';
+import { LDP } from '../../src';
+import { describeIf, ResourceHelper } from '../util/TestHelpers';
```

Comment:
Going to do a full review later but can already say this (annoying request): can you rewrite the integration test to not use `ResourceHelper` and use fetch instead? The reason is that we want to eventually™️ (#588) get rid of the test framework that's behind that since there are issues with it. You can see examples of integration tests with fetch in the `DynamicPods`, `PodCreation`, or `Subdomains` tests. Or if you prefer something like `supertest` that is being used in the `Middleware` integration test is also fine.